src/backend.github.js

```diff
@@ -383,7 +383,13 @@ var _ = Mavo.Backend.register($.Class({
 				ret.filename = defaults.filename;
 			}
 
-			ret.filepath = path.join("/") || defaults.filepath || "";
+			// If an author provided backend metadata, use them
+			// since they have higher priority
+			for (const prop in ret) {
```

Comment:
One issue with this is that it only iterates over properties already defined on `ret`, but a common case would be to define things via attributes and NOT in the URL, so they may not exist as keys on `ret`.